tests/test_datetime_utils.py

```diff
@@ -47,14 +49,14 @@ def test_yesterday(self):
         self.assertEqual(cl_yesterday.year, dt_yesterday.year)
 
     def test_today(self):
-        dt_today = datetime.today()
+        dtu_today = dateutils_today()
         cl_today = today()
         self.assertNotEqual(cl_today, None)
         self.assertEqual(cl_today.minute, 0)
         self.assertEqual(cl_today.hour, 0)
```

Comment:
what is the minutes, hour. of  dtu_today?

Reply:
added the test to test with `dtu_today`'s minute and hour

Reply:
it's returning `00:00:00`